### PR TITLE
[bitbay] - api upgrade

### PR DESCRIPTION
**Pull request by npomfret**
* upgraded `cancelOrder`  to new api
 * upgraded `fetchBalance`  to new api

@kroitor I believe the old private api is no longer used here

### CODE REVIEW

File: js/bitbay.js [hunk 1]
```diff
@@ -933,10 +931,24 @@ module.exports = class bitbay extends Exchange {
     }
 
     async cancelOrder (id, symbol = undefined, params = {}) {
+        if (!params['side']) {
+            throw new ExchangeError (this.id + ' parameter named side is required');
+        }
+        if (!params['price']) {
```
Comment (kroitor) [inline js/bitbay.js]: Why would they require it... :\

Comment (npomfret) [reply js/bitbay.js]: I know.  I tested without it and it doesn't throw an error.

Comment (npomfret) [reply js/bitbay.js]: If it's too annoying for people, we can always look up the price and amount with another API query.

Comment (kroitor) [reply js/bitbay.js]: @npomfret nope, no worries, it's better to have it required explicitly than to send another implicit request. I'll just add a few minor edits for Python. Thx! )

**Pull request merged**